I'm sorry, but errors were detected in your script. Please correct the
errors listed below, and try again.


The label good_p4 is defined twice, at File "game/phase_4_dinner.rpy", line 186:
            label good_p4:
and File "game/phase_4_dinner.rpy", line 594:
            label good_p4:


The label bad_p4 is defined twice, at File "game/phase_4_dinner.rpy", line 199:
            label bad_p4:
and File "game/phase_4_dinner.rpy", line 607:
            label bad_p4:


Ren'Py Version: Ren'Py 7.4.9.2142
Sun Mar 27 16:03:07 2022
